frameworks/elastic/universe/config.json

```diff
@@ -521,7 +530,7 @@
         "http_content_type_required": {
           "description": "Enables or disables strict checking and usage of the Content-Type header for all requests with content, defaults to false.",
           "type": "boolean",
-          "default": false
+          "default": true
```

Comment:
Is this a requirement for the base-tech update?

Reply:
If there is no way to disable it, then we should consider adding a note to the description -- and remember to add a note to the release notes.

Reply:
ok we will add it @ryadav88